atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/transaction/TransactionConfig.java

```diff
@@ -32,4 +32,8 @@ public long getLockAcquireTimeoutMillis() {
         return AtlasDbConstants.DEFAULT_TRANSACTION_LOCK_ACQUIRE_TIMEOUT_MS;
     }
 
+    @Value.Default
+    public int getThresholdForLoggingLargeNumberOfTransactionLookups() {
+        return AtlasDbConstants.THRESHOLD_FOR_LOGGING_LARGE_NUMBER_OF_TRANSACTION_LOOKUPS;
```

Comment:
As mentioned above it would be cool for this to be a map of table-name to value or something like that (where an empty map implies all tables use the default.